nexus/tests/integration_tests/endpoints.rs

```diff
@@ -790,6 +805,25 @@ lazy_static! {
             allowed_methods: vec![AllowedMethod::Get],
         },
 
+        VerifyEndpoint {
+            url: &*SSH_KEYS_URL,
+            visibility: Visibility::Protected,
+            allowed_methods: vec![
+                AllowedMethod::Get,
+                AllowedMethod::Post(
+                    serde_json::to_value(&*DEMO_SSH_KEY_CREATE).unwrap()
+                ),
+            ],
+        },
+        VerifyEndpoint {
+            url: &*DEMO_SSH_KEY_URL,
```

Comment:
I suspect you'll need to add a `SetupReq` in unauthorized.rs that actually creates this demo SSH key.  Otherwise, when it tries to hit this URL with a privileged user, it'll get a 404, but it expects to get a 200.

Reply:
Ok, done in af0b7691. However, there are other `VerifyEndpoint`s without corresponding `SetupReq`s, e.g., `DEMO_INSTANCE_NIC_CREATE`. I guess the assumption is that the endpoints are run in order, so after a `POST` it's ok to do `GET` and `DELETE`. Should those others be updated as well?